osu.Game/Screens/Multi/Lounge/Components/RoomsContainer.cs

```diff
@@ -88,6 +96,7 @@ private void addRooms(IEnumerable<Room> rooms)
             {
                 roomFlow.Add(new DrawableRoom(room)
                 {
+                    DuplicateRoom = DuplicateRoom,
```

Comment:
is there some reason I'm missing as to why this delegate is passed down a level instead of declared locally here?

Reply:
The problem was getting access to `LoungeSubScreen.Open`, I was unsure on the best way to do that so I passed the delegate down 2 levels

My second idea was to cache `LoungeSubScreen` and resolve it in `RoomsContainer`, would that be better?

Reply:
caching would be preferable as far as I'm concerned, yeah.